event-land/index.html

```diff
@@ -8,10 +8,41 @@
       name="viewport"
       content="initial-scale=1.0, user-scalable=no, width=device-width"
     />
+    <script
+      type="module"
+      src="https://unpkg.com/ionicons@7.1.0/dist/ionicons/ionicons.esm.js"
+    ></script>
+    <script
+      nomodule
+      src="https://unpkg.com/ionicons@7.1.0/dist/ionicons/ionicons.js"
+    ></script>
     <link rel="stylesheet" href="css/style.css" />
   </head>
   <body>
-    <nav class="top-bar">Top bar</nav>
+    <nav class="top-bar">
+      <div class="top-bar__business">
+        <span>small business</span>
+        <span>company</span>
+      </div>
+      <ul class="top-bar__actions">
+        <li><a href="#">contact us</a></li>
+        <li class="dropdown">
+          <span> (423) 323-3382</span>
+          <ion-icon
+            name="chevron-down-outline"
+            class="dropdown__icon"
+          ></ion-icon>
+          <ul class="dropdown__content">
```

Comment:
Why you use  BEM (Block Element Modifier)?

Reply:
Because that is what we have defined in code convention page, we should use BEM class naming convention ;0